app/assets/javascripts/discourse/app/templates/navigation/category.hbs

```diff
@@ -2,8 +2,7 @@
 
 <section class="category-heading">
   {{#if this.category.uploaded_logo.url}}
-    <CdnImg @src={{this.category.uploaded_logo.url}} @class="category-logo" @width={{this.category.uploaded_logo.width}} @height={{this.category.uploaded_logo.height}} @alt="" />
-
+    <CategoryLogo @category={{this.category}} @class="category-logo" />
```

Comment:
I don't see the "category-logo" class being used either, that also can be omitted.